### DISCUSSION TITLE
How to ignore error for python

### DISCUSSION THREAD

**Discussion by krishnagogada** [mkslanc/ace-linters#123] [Q&A] [answered]
I am using the latest version. I want to ignore some errors in the Python language.
I explored [ruff](https://github.com/astral-sh/ruff), and there I found the `ignore` key. So how can I pass this `ignore` key? Or else is there any other way like for typescript we have `errorCodesToIgnore` key.

**Comment by mkslanc** [comment, marked as answer]
hey @krishnagogada, for python you could use same options - 
```typescript
export interface FilterDiagnosticsOptions {
    errorCodesToIgnore?: string[],
    errorCodesToTreatAsWarning?: string[]
    errorCodesToTreatAsInfo?: string[],
    errorMessagesToIgnore?: RegExp[],
    errorMessagesToTreatAsWarning?: RegExp[]
    errorMessagesToTreatAsInfo?: RegExp[]
}
```

or pass ignore's param to configuration similar to [this](https://github.com/mkslanc/ace-linters/blob/main/packages/demo/webworker-lsp/demo.ts#L135)
